File: ifthenelseif.erl
% ifthenelseif.erl
-module(ifthenelseif).
-export([main/0]).

main() ->
    X = 5,
    if X =:= 5 ->
        io:format("x is 5~n");
    X < 10 ->
        io:format("x is less than 10~n")
    end.

File: ifthenelseif.zig
// ifthenelseif.zig
const std = @import("std");

pub fn main() void {
    const x: i32 = 5;
    if (x == 5) {
        std.debug.print("x is 5\n", .{});
    } else if (x < 10) {
        std.debug.print("x is less than 10\n", .{});
    }
}
